brainstormforce/ultimate-addons-for-gutenberg
# Fix: Modal trigger icon bottom space issue

### Description

- https://brainstormforce.slack.com/archives/CAQ2TSA6A/p1681164807429969

### Screenshots

- https://share.getcloudapp.com/7Kuzj4bp

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Modal.
- Choose trigger as a icon. (In FSE as well on post/pages)
- Verify the unwanted bottom space.

### Checklist:
